docs/handlers/TSDBHandler.md

```diff
@@ -52,26 +55,43 @@ We don't automatically make the metrics via **mkmetric**, so either set
 ```
 set tsd.core.auto_create_metrics = true
 ```
-in your [OpenTSDB configuration](http://opentsdb.net/docs/build/html/user_guide/configuration.html)
+in your [OpenTSDB configuration]
+(http://opentsdb.net/docs/build/html/user_guide/configuration.html)
 or you can run with the null handler and log the output
-and extract the key values to mkmetric yourself  
+and extract the key values to mkmetric yourself
 
 - enable it in `diamond.conf` :
 
 `    handlers = diamond.handler.tsdb.TSDBHandler
 `
+We will automatically split bigger packages of metrics. You have to set
+tsd.http.request.enable_chunked = true
+and if pakackes get to big you will also need to adjust
+tsd.http.request.max_chunk which is 4096 by default. Learn more
+[here](http://opentsdb.net/docs/build/html/user_guide/configuration.html).
+
+In your diamond.conf:
+
+You can define how many packages you want to send in one package by setting
+batch<=1. We dont recommend 1 as it may have bigger impact on your CPU.
+
+Compression can be enabled by setting compression to 1-9 while 1 is low and 9 is
+high.
 
 
 #### Options
 
-Setting | Default | Description | Type
---------|---------|-------------|-----
-format | {Collector}.{Metric} {timestamp} {value} hostname={host}{tags} |  | str
-get_default_config_help |  | get_default_config_help |
-host |  |  | str
-port | 1234 |  | int
-server_error_interval | 120 | How frequently to send repeated server errors | int
-tags |  | Tags to be added to each metric. They should be key/value pairs. Example : tags = environment=test,datacenter=north| str
-timeout | 5 |  | int
-cleanMetrics| True | Extract tag values from known collectors and make the metrics more OpenTSDB style | bool
-skipAggregates| True | Only has effect when cleanMetrics is true. Then the metrics that are considered aggregates are removed | bool
+| Setting                 | Default | Description                                                                                            | Type |
```

Comment:
What do I need to do, to get `build_doc.py` working? Why does it need the diamond config file?